src/stylesheets/objects/_content-layouts.scss

```diff
@@ -68,15 +68,12 @@ header .container {
 }
 
 .navigation--main .container {
-  max-width: 732px; // Reduced to accommodate current grid
-  padding: 0;
-  margin: 0 $base-spacing-unit-small;
+  border-bottom: solid 1px $grey-3;
```

Comment:
This is causing the border to bleed to the left and right of mobile screens, it might be better to add style to the unordered-list inside `.navigation--main` instead.